src/models/shopSchema.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
```

Comment:
Normally you match the file name exactly to the thing you're exporting. So in this case `ShopSchema`